app/mixins/scroll-reset.js

```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 
 export default Ember.Mixin.create({
   beforeModel: function () {
-    this._super(...arguments);
     window.scrollTo(0, 0);
+    return this._super(...arguments);
```

Comment:
Does this make a difference perception-wise?

Reply:
You mean for users? This is the bug that I was talking about the other day - if we don't return the result of the `_super` call, we may miss something that one of the routes did. For example if the `_super` call returns a rejected promise, if we don't return it here, it will get ignored. I'm not sure which places could be affected by this, but not returning here could lead to a confusing behaviour (in fact it took me quite some debugging time to figure out why auth doesn't work as it should for some routes)